docs/style-guide.rst

```diff
@@ -188,8 +188,8 @@ Event Definitions and Event Emitters
 Yes::
 
     event LongAndLotsOfArgs(
-        adress sender,
-        adress recipient,
+        address sender,
```

Comment:
This is gold!

Reply:
Especially that this was found in the style guide.